frontend/app/mocks/raoidc.server.ts

```diff
@@ -123,7 +123,7 @@ async function generateIdToken(clientPublicKey: string, serverPrivateKey: string
     aud: 'CDCP',
     iss: 'GC-ECAS-MOCK',
     sid: '00000000-0000-0000-0000-000000000000',
-    sub: '00000000-0000-0000-0000-000000000000',
+    sub: '76c48130-e1d4-4c2f-8dd0-1c17f9bbb4f6',
```

Comment:
This is not really future-proof. Please add a `TODO` that says something like "allow caller to specify sub as a parameter"

Reply:
Maybe a nitpick, but although a `TODO` has been added but it's not really clear what to do. As a developer, I don't know what "this method" means.